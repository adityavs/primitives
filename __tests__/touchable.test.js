/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';

import { Touchable } from '../index';

describe('<Touchable>', () => {
  it('should render a touchable component', () => {
    const tree = renderer.create(<Touchable onPress={() => null} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render a disabled touchable component', () => {
    const tree = renderer
      .create(<Touchable onPress={() => null} isDisabled />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
