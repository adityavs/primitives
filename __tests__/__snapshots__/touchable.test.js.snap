// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Touchable> should render a disabled touchable component 1`] = `
<View
  accessible={true}
  focusable={true}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
/>
`;

exports[`<Touchable> should render a touchable component 1`] = `
<View
  accessible={true}
  focusable={true}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
/>
`;
